vm/zod/src/params/zLoadStateParams.js

```diff
@@ -0,0 +1,17 @@
+import { zHex } from '../common/index.js'
+import { z } from 'zod'
+/**
+ * Zod validator for a valid load state action
+ */
+
+const AccountStorage = z.object({
+	nonce: z.bigint(),
+	balance: z.bigint(),
+	storageRoot: z.instanceof(Uint8Array),
+	codeHash: z.instanceof(Uint8Array),
```

Comment:
```suggestion
	storageRoot: zHex,
	codeHash: zHex,
```
To stay consistent with viem the tevm api (when public) always uses hex strings instead of bytes